packages/contracts/package.json

```diff
@@ -16,8 +16,9 @@
   "types": "dist/index",
   "license": "MIT",
   "scripts": {
-    "all": "yarn clean && yarn build && yarn test && yarn lint:fix && yarn lint",
-    "build": "./scripts/build.sh",
+    "all": "yarn build && yarn test && yarn lint:fix && yarn lint",
```

Comment:
Why did you remove `clean`?

Reply:
@snario I figure if its in the `yarn build` script, its redundant to have it in `yarn all`, but happy to revert if you'd like. lmk